modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -891,25 +913,30 @@ struct Infer: public cv::detail::KernelTag {
         // meta order.
         GAPI_Assert(uu.params.input_names.size() == in_metas.size()
                     && "Known input layers count doesn't match input meta count");
-        for (auto &&it : ade::util::zip(ade::util::toRange(uu.params.input_names),
-                                        ade::util::toRange(in_metas))) {
-            const auto &input_name = std::get<0>(it);
-            auto       &&ii = uu.inputs.at(input_name);
-            const auto & mm = std::get<1>(it);
 
-            configureInputInfo(ii, mm);
-            if (uu.params.layer_names_to_reshape.find(input_name) !=
-                uu.params.layer_names_to_reshape.end()) {
-                configureInputReshapeByImage(ii, mm, input_reshape_table);
+        // NB: Configuring input precision and network reshape must be done
+        // only for loadNetwork way.
```

Comment:
`only in the LoadNetwork case` is maybe a better wording

Reply:
Done